core/src/test/scala/org/apache/spark/io/ChunkedByteBufferSuite.scala

```diff
@@ -56,6 +56,13 @@ class ChunkedByteBufferSuite extends SparkFunSuite {
     assert(chunkedByteBuffer.getChunks().head.position() === 0)
   }
 
+  test("SPARK-24107: writeFully() write buffer which is larger than bufferWriteChunkSize") {
+    val chunkedByteBuffer = new ChunkedByteBuffer(Array(ByteBuffer.allocate(80 * 1024 * 1024)))
```

Comment:
Can you configure `bufferWriteChunkSize` explicitly?

Reply:
I have modified.Please check